.github/workflows/pytest.yaml

```diff
@@ -0,0 +1,24 @@
+name: Run pytest
```

Comment:
To jest zacna sprawa która spowoduje że github za ciebie odpali testy za każdym razem jak jakieś zmiany będą wypchane. Warto w [ustawieniach ustawić](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/managing-a-branch-protection-rule#creating-a-branch-protection-rule) że wszystkie testy muszą przejść zanim można wciągnąć zmiany

Reply:
można w podobny sposób mu kazać coś robić co jakiś czas, np. wysłać ci maila z tym co masz brać

Reply:
black magic

Reply:
owszem :D Ale jakże przydatna... :D